meditate/views.py

```diff
@@ -18,7 +18,7 @@
 
 
 # See your keys here: https://dashboard.stripe.com/account/apikeys
-stripe.api_key = os.environ.get('STRIPE_SECRET_KEY', '')
+stripe.api_key = os.environ.get('STRIPE_SECRET_KEY', 'asa')
```

Comment:
Why populate this?  The check to make sure it's set on run would never work like this.